mxnet-bot/LabelBotPredict/elastic_beanstalk_server/test_sentenceparse.py

```diff
@@ -0,0 +1,69 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import requests
+import unittest
+import boto3
+import pandas as pd
+from botocore.exceptions import ClientError
+from botocore.exceptions import NoCredentialsError
+from SentenceParser import SentenceParser
+from pandas.util.testing import assert_frame_equal
+# some version issue
+try:
+    from unittest.mock import patch
+except ImportError:
+    from mock import patch
+
+# test coverage: 88%
+class TestSentenceParser(unittest.TestCase):
+
+	def setUp(self):
+		self.sp = SentenceParser()
+		self.sp.data = pd.DataFrame([{'id': 11925, 'title': "issue's title", 
+									  'body': " bug ``` import pandas``` ## Environment info", 
+									  'labels': ['Doc']}])
+
+	def tearDown(self):
```

Comment:
What is the purpose of this function, shall we remove?

Reply:
+1

Reply:
Removed